dlls/game.cpp

```diff
@@ -92,7 +92,7 @@ cvar_t sk_scientist_heal3 = { "sk_scientist_heal3", "0", 0, 0.0f, NULL };
 void EXT_FUNC GameDLLInit()
 {
 	g_bIsCzeroGame = UTIL_IsGame("czero");
-   g_bEnableCSBot = g_bIsCzeroGame || ENG_CHECK_PARM ("-bots", NULL) != 0;
+	g_bEnableCSBot = g_bIsCzeroGame || ENG_CHECK_PARM ("-nobots", NULL) == 0;
```

Comment:
well that is for my private use & isn't supposed to be in the pull request. only the first commit is :(